pkg/routes/index.go

```diff
@@ -1,8 +1,30 @@
 package routes
 
+import (
+	"fmt"
+
+	"github.com/AngelVI13/fiber-investigation/pkg/database"
+)
+
 func (r *Router) HandleIndex(c *Ctx) error {
 	data := c.FlashData()
 	data["Title"] = "Keyword storage"
 
+	businesskwds, err := database.BusinessKeywords(r.db)
+	if err != nil {
+		return c.WithError(fmt.Sprintf(
+			"error while fetching business keywords: %v", err),
+		).Render(KeywordsView, data)
```

Comment:
```suggestion
		).Render(IndexView, data)
```

same for other error

Reply:
fixed